lib/xlsx/xform/table/filter-column-xform.js

```diff
@@ -4,6 +4,8 @@ const ListXform = require('../list-xform');
 const CustomFilterXform = require('./custom-filter-xform');
 const FilterXform = require('./filter-xform');
 
+const undefinedOrResult = (test, result) => (typeof test === 'undefined' ? undefined : result);
```

Comment:
```suggestion
const result = (test, result) => (typeof test === 'undefined' ? undefined : result);
```
I think better keeping it simple to `result`

Reply:
"result" is too generic and does not help to understand the purpose of the function?